src/test/java/ioanadroflorea/tests/Tema6/HerokuAlerts.java

```diff
@@ -0,0 +1,94 @@
+package ioanadroflorea.tests.Tema6;
+
+import ioanandroflorea.driver.BrowserManager;
+import ioanandroflorea.utils.FileUtils;
+import org.openqa.selenium.*;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.interactions.Actions;
+
+import java.io.File;
+import java.io.IOException;
+
+public class HerokuAlerts {
+    static ChromeDriver driver = null;
+
+    public static void main(String[] args) throws IOException, InterruptedException, NoSuchElementException {
+        navigateToHerokuHomePage();
+        openAlertsPage();
+        openFirstAlert();
+        acceptFirstAlert();
+        openSecondAlert();
+        acceptSecondAlert();
+        openThirdAlert();
+        completeThirdAlert();
+        scrollToSubmitButton();
```

Comment:
Here is the line where we are getting a NoSuchElementException due to the fact that there is no submit button on this page. Please add this line in the catch block too. Thanks!